SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityStartup.java

```diff
@@ -123,7 +139,15 @@ void checkSchedulerForInconsistentState() {
         case ACTIVE:
         case SYSTEM_COOLDOWN:
         case DEPLOYING_TO_UNPAUSE:
-          checkActiveRequest(requestWithState, deployKeyToPendingTaskId, now);
+          if(checkActiveRequest(requestWithState, deployKeyToPendingTaskId, now)) {
+            numPendingStartupRequests++;
+            double startUpTaskPct = (numPendingStartupRequests * 100.0 / totalStartupRequests);
```

Comment:
another nit on naming, not working with any tasks here, just requests and pending requests